ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/PlatformCvesOverviewPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Card,
+    CardBody,
+} from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+import useURLStringUnion from 'hooks/useURLStringUnion';
+import useURLPagination from 'hooks/useURLPagination';
+import useURLSearch from 'hooks/useURLSearch';
+import { getHasSearchApplied } from 'utils/searchUtils';
+
+import EntityTypeToggleGroup from '../../components/EntityTypeToggleGroup';
+import { parseWorkloadQuerySearchFilter } from '../../utils/searchUtils';
+import { platformEntityTabValues } from '../../types';
+
+import CVEsTableContainer from './CVEsTableContainer';
+import ClustersTableContainer from './ClustersTableContainer';
+
+function PlatformCvesOverviewPage() {
+    const [activeEntityTabKey] = useURLStringUnion('entityTab', platformEntityTabValues);
+    const { searchFilter } = useURLSearch();
+    const pagination = useURLPagination(20);
+
+    // TODO - Need an equivalent function implementation for filter sanitization for Platform CVEs
+    const querySearchFilter = parseWorkloadQuerySearchFilter(searchFilter);
+    const isFiltered = getHasSearchApplied(querySearchFilter);
+
+    function onEntityTabChange() {
+        pagination.setPage(1);
+        // TODO - set default sort here
+    }
+
+    // TODO - needs to be connected to a query
+    const entityCounts = {
+        CVE: 0,
+        Cluster: 0,
+    };
+
+    const filterToolbar = <></>;
```

Comment:
I'm going to do my best to ignore filtering and the filter bar until advanced filters is complete, so I've omitted the component here.